test/parse-argv-test.js

```diff
@@ -154,25 +136,10 @@ describe('parseArgv()', function() {
       expect(result.connect).to.equal(true);
     });
 
-    it('can be explicitly set to true', function() {
-      var result = _parseArgv(['--connect']);
-      expect(result.connect).to.equal(true);
-    });
-
-    it('can be explicitly set to true via alias', function() {
-      var result = _parseArgv(['--ct']);
-      expect(result.connect).to.equal(true);
-    });
-
     it('can be set to false', function() {
       var result = _parseArgv(['--no-connect']);
       expect(result.connect).to.equal(false);
     });
```

Comment:
@marcoow the test case above checks the `--no-connect` -> `connect: false` conversion